redis/tests/test_cache.rs

```diff
@@ -32,7 +32,7 @@ macro_rules! assert_invalidate {
 
 // Basic testing should work with both CacheMode::All and CacheMode::OptIn if commands has called cache()
 #[rstest]
-#[case::tokio(RuntimeType::Tokio)]
+#[cfg_attr(feature = "tokio-comp", case::smol(RuntimeType::Tokio))]
```

Comment:
```suggestion
#[cfg_attr(feature = "tokio-comp", case::tokio(RuntimeType::Tokio))]
```

problem also exists in the file

Reply:
Thanks for the catch!